pyveg/src/download_modules.py

```diff
@@ -126,11 +127,25 @@ def prep_data(self, date_range):
         -------
         url_list:  a list of URLs from which zipfiles can be downloaded from GEE.
         """
-        region = get_region_string(self.coords, self.region_size)
+        region = get_region_string(self.bounds)
```

Comment:
I know I wrote this (!) but:

1) Need to remove all the debug statements (minor)
2) Need to decide whether to go with hardcoding BNG (EPSG:27700) throughout. Or make this a parameter defined in the config. If the projection is not specified, it defaults to WGS84 Long/Lat
https://developers.google.com/earth-engine/apidocs/ee-geometry
3) Note that the `region` variable is not used anymore.

Reply:
Debug statements and region now removed.

I've made the projection now configurable and defaulting to BNG if not configuration is given. The issue I see atm is that a lot of the name formatting now is assuming BNG, and needs to be adapted if is not (it can look weird in lat long). I think options for this can be added later on once we are tyding out the code given that we only care about BNG for this project.